### PR TITLE
Minor graph API cleanup

### PR DESCRIPTION
**Pull request by tbennun**

### CODE REVIEW

**Comment by tbennun** [comment]
Depends on #262

File: dace/sdfg.py [hunk 1]
```diff
@@ -4658,7 +4658,7 @@ class skips the process.
     if class_override is None:
         clazz = Config.get("optimizer", "interface")
 
-    if clazz == "" or clazz == False:
+    if clazz == "" or clazz is False:
```
Comment (definelicht) [inline dace/sdfg.py:4661]: `cls` not `clazz` omg :D

Comment (tbennun) [reply dace/sdfg.py:4661]: I'll make a pass in the next branch

Comment (definelicht) [reply dace/sdfg.py:4661]: It's _really_ not important ;-)

**Pull request merged**